### ISSUE TITLE
CentOS 6/7 packages for openssl 1.1.1 

### ISSUE THREAD

**Issue by bixelsnl** [openresty/openresty#572]
OpenSSL 1.1.1 was implemented in https://github.com/openresty/openresty/issues/456 . There is a comment that it should be a new ticket to create the CentOS repo packages. However, I cannot find such a ticket. So here it is.

Please let us know what we can do to help. If there is any documentation/guidelines on doing the actual packaging, we would be willing to contribute.

**Comment by bixelsnl** [comment]
Just to keep track of some more information I stumbled upon regarding this. Someone is working on the build scripts for these in https://github.com/openresty/openresty-packaging/pull/36 . They build against the latest stable release `1.15.8.2`.

Also tagging https://github.com/openresty/openresty/issues/530 for reference - it is not specifically about CentOS packages, but is related I would say.

**Comment by agentzh** [comment]
Yeah, we will take care of this very soon. Sorry for the delay on our side.

**Comment by bixelsnl** [comment]
No problem, thanks for taking care of this. Not to rush you, but do you have any ETA? We are actually in the process of building our own packages and setting up our own repo. The idea is that we can provide our 700+ servers with TLS 1.3 soon. But of course, it makes no sense for us to invest more time here, if you are releasing your package next week :) On the other hand, if it will take a month, we will probably proceed with our plan.

Again, no pressure; but an ETA would really help us in our decisionmaking.

**Comment by bixelsnl** [comment]
I am seeing some openrest 111 packages in the CentOS repo. Does this mean that these packages are now generally available, and we can stop using our own custom built packages?

**Comment by bixelsnl** [comment]
To be clear: technically you are right, there are now openssl111 packages. But I still cannot use the official repo because `openresty` is not compiled against that new `openssl111` package:

```
$ rpm -ivh openresty-1.15.8.3-1.el6.x86_64.rpm 
warning: openresty-1.15.8.3-1.el6.x86_64.rpm: Header V4 RSA/SHA1 Signature, key ID d5edeb74: NOKEY
error: Failed dependencies:
        openresty-zlib >= 1.2.11-3 is needed by openresty-1.15.8.3-1.el6.x86_64
        openresty-openssl >= 1.1.0l-1 is needed by openresty-1.15.8.3-1.el6.x86_64
        openresty-pcre >= 8.44-1 is needed by openresty-1.15.8.3-1.el6.x86_64
```

Will await the release of the new `openresty-1.17.8.1` package.
